vulcanai/models/layers.py

```diff
@@ -301,6 +301,7 @@ def __init__(self, in_channels, out_channels, bias=False):
 
     def forward(self, input):
```

Comment:
yeah can make sure that this class is hidden so it doesn't automatically get used because it's not close to being ready and it's a bit of a vestigial organ for this file? Thank you

Reply:
done